cva6/sim/cva6.py

```diff
@@ -431,7 +431,7 @@ def run_assembly(asm_test, iss_yaml, isa, target, mabi, gcc_opts, iss_opts, outp
     base_cmd = parse_iss_yaml(iss, iss_yaml, isa, target, setting_dir, debug_cmd)
     cmd = get_iss_cmd(base_cmd, elf, target, log)
     logging.info("[%0s] Running ISS simulation: %s" % (iss, cmd))
-    run_cmd(cmd, 300, debug_cmd = debug_cmd)
+    run_cmd(cmd, 1200, debug_cmd = debug_cmd)
```

Comment:
Please don't modify this value. Note: removing this modification makes commit 1ddadce unnecessary

Reply:
Addressed in PR #1427 which superseded the present one (#1424).